### PR TITLE
3.5.0

### PR DESCRIPTION
**Pull request by franmomu**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ a release.
 ---
 
 ## [Unreleased]
+
+## [3.5.0] - 2022-01-10
 ### Added
 - SoftDeleteable: Support to use annotations as attributes on PHP >= 8.0.
```
Comment (phansys) [inline CHANGELOG.md:25]: Please, fix the indentation.

Comment (franmomu) [reply CHANGELOG.md:25]: hmm how?

Comment (phansys) [reply CHANGELOG.md:25]: Forget about it. I can confirm the indentation is fine by checking out the commit locally.
Maybe there is a visual issue with the Github UI:
image

**Review by phansys — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (franmomu) [reply CHANGELOG.md:25]: 🤔 looks like it's when there is no space at the beginning, it adds like a padding to be able to show the ➕ icon on hover.

**Pull request merged**